packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -203,7 +203,7 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
       model.getColumnHeaderParentGroup(modelIndex, 0) === undefined &&
       !(isExpandableGridModel(model) && model.hasExpandableRows);
     const isColumnFrozen = model.isColumnFrozen(modelIndex);
-    const isColumnSortable = column.isSortable;
+    const isColumnSortable = column.isSortable ?? false;
```

Comment:
We want to default to `true` if the `isSortable` property isn't present. Otherwise we won't be able to sort at all on Enterprise!